data/test-cases/plugins/gitlab/context-menu-feature/Create_gitlab_issue.md

```diff
@@ -0,0 +1,49 @@
+---
+# (Required) Ensure all values are filled up
+name: "Create a Gitlab issue from the context menu on MM."
+status: Active
+priority: Normal
+folder: Context-menu-feature
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Hover cursor over a non-system message and click on the `message-actions` then select `Create Gitlab Issue` on MM.
```

Comment:
What do you mean by a non-system message, use appropriate term for this.

Reply:
By `non-system message` i meant the messages that were added by a users account, it was mentioned in the PR as `non-system message` that why I followed this naming format,  but not i have changed it by `user message`.